pkg/cvo/cvo_scenarios_test.go

```diff
@@ -70,12 +70,12 @@ func setupCVOTest(payloadDir string) (*Operator, map[string]runtime.Object, *fak
 	})
 
 	o := &Operator{
-		namespace:                   "test",
-		name:                        "version",
+		namespace: "test",
```

Comment:
I think this is backsliding on Go 1.13's formatting:

```console
$ git --no-pager log --oneline -1
33c473e (HEAD, origin/pr/309) Further refactoring of CVO in prep to move verify package
$ go1.13.5 fmt ./pkg/cvo/
pkg/cvo/cvo_scenarios_test.go
$ git diff
diff --git a/pkg/cvo/cvo_scenarios_test.go b/pkg/cvo/cvo_scenarios_test.go
index 7c978ba..712e97a 100644
--- a/pkg/cvo/cvo_scenarios_test.go
+++ b/pkg/cvo/cvo_scenarios_test.go
@@ -70,12 +70,12 @@ func setupCVOTest(payloadDir string) (*Operator, map[string]runtime.Object, *fak
 	})
 
 	o := &Operator{
-		namespace: "test",
-		name:      "version",
+		namespace:                   "test",
+		name:                        "version",
 		enableDefaultClusterVersion: true,
-		queue:    workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "cvo-loop-test"),
-		client:   client,
-		cvLister: &clientCVLister{client: client},
+		queue:                       workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "cvo-loop-test"),
+		client:                      client,
+		cvLister:                    &clientCVLister{client: client},
 	}
 
 	dynamicScheme := runtime.NewScheme()
@@ -159,7 +159,7 @@ func TestCVO_StartupAndSync(t *testing.T) {
 		},
 		Status: configv1.ClusterVersionStatus{
 			History: []configv1.UpdateHistory{
-			// empty because the operator release image is not set, so we have no input
+				// empty because the operator release image is not set, so we have no input
 			},
 			Conditions: []configv1.ClusterOperatorStatusCondition{
 				{Type: configv1.OperatorAvailable, Status: configv1.ConditionFalse},
@@ -430,7 +430,7 @@ func TestCVO_StartupAndSyncUnverifiedPayload(t *testing.T) {
 		},
 		Status: configv1.ClusterVersionStatus{
 			History: []configv1.UpdateHistory{
-			// empty because the operator release image is not set, so we have no input
+				// empty because the operator release image is not set, so we have no input
 			},
 			Conditions: []configv1.ClusterOperatorStatusCondition{
 				{Type: configv1.OperatorAvailable, Status: configv1.ConditionFalse},
@@ -691,7 +691,7 @@ func TestCVO_StartupAndSyncPreconditionFailing(t *testing.T) {
 		},
 		Status: configv1.ClusterVersionStatus{
 			History: []configv1.UpdateHistory{
-			// empty because the operator release image is not set, so we have no input
+				// empty because the operator release image is not set, so we have no input
 			},
 			Conditions: []configv1.ClusterOperatorStatusCondition{
 				{Type: configv1.OperatorAvailable, Status: configv1.ConditionFalse},
```

Can we leave those indents alone to avoid churning the code?

Reply:
Fixed. Not purposeful but I believe due to new editor options which I have backed out.

Reply:
Fixed with 33c473e -> be97bc5, thanks.